.github/workflows/test_tests.yml

```diff
@@ -28,9 +28,9 @@ jobs:
       PLAYGROUND_PASSWORD: ${{ secrets.PLAYGROUND_PASSWORD }}
       PLAYGROUND_ID: ${{ secrets.PLAYGROUND_ID }}
       DOCASSEMBLE_DEVELOPER_API_KEY: ${{ secrets.DOCASSEMBLE_DEVELOPER_API_KEY }}
-      BRANCH_PATH: main
+      BRANCH_PATH: move_tests_internal
```

Comment:
I think we need to change this to be more like our action workflow so that the branch and repo address get created automatically based on the test. As soon as we delete this branch, these tests will fail. See https://github.com/SuffolkLITLab/ALKiln/blob/releases/v4/action.yml#L55-L56.

Reply:
Tests haven't quite finished yet, but looks like they work well! https://github.com/SuffolkLITLab/ALKiln/actions/runs/3321851857/jobs/5490068479#step:5:21 gets the right repo URL and the step above prints the right branch.